src/Composer/ComposerPatchesConfigurationUpdater.php

```diff
@@ -56,4 +71,15 @@ public function updateComposerJsonAndPrint(string $composerJsonFilePath, array $
         $composerJsonFileContents = Json::encode($composerJson, Json::PRETTY);
         FileSystem::write($composerJsonFilePath, $composerJsonFileContents);
     }
+
+    /**
+     * @param mixed[] $composerExtraPatches
+     */
+    public function updatePatchesFileJsonAndPrint(string $patchesFilePath, array $composerExtraPatches): void
+    {
+        $patchesFileJson = $this->updatePatchesFileJson($patchesFilePath, $composerExtraPatches);
+
+        $patchesFileContents = Json::encode($patchesFileJson, Json::PRETTY);
+        FileSystem::write($patchesFilePath, $patchesFileContents);
```

Comment:
pass null on 3rg argument to keep permission

```suggestion
        FileSystem::write($patchesFilePath, $patchesFileContents, null);
```